### PR TITLE
translate tree component

### PR DESCRIPTION
**Pull request by GrayChoi**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!
- [X] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
- [X] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
- [X] Rebase before creating a PR to keep commit history clear.
- [X] Add some descriptions and refer relative issues for you PR.n
#1471

### CODE REVIEW

File: components/tree/demo/draggable.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+title: 
+  zh-CN: 拖动示例
+  en-US: draggable example
 ---
 
+##zh-CN
+
 将节点拖拽到其他节点内部或前后。
 
+##en-US
```
Comment (afc163) [inline components/tree/demo/draggable.md]: `##en-US` => `## en-US`

File: components/tree/demo/draggable.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 2
-title: 拖动示例
+title: 
+  zh-CN: 拖动示例
+  en-US: draggable example
 ---
 
+##zh-CN
```
Comment (afc163) [inline components/tree/demo/draggable.md]: `##zh-CN` => `## zh-CN`

File: components/tree/demo/draggable.md [hunk 3]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 2
-title: 拖动示例
+title: 
+  zh-CN: 拖动示例
+  en-US: draggable example
```
Comment (afc163) [inline components/tree/demo/draggable.md]: Just `Draggable`

File: components/tree/demo/draggable.md [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
 ---
 
+##zh-CN
+
 将节点拖拽到其他节点内部或前后。
 
+##en-US
+
+drag treeNode to insert after the other treeNode or insert into the other parent TreeNode.
```
Comment (afc163) [inline components/tree/demo/draggable.md]: `Drag`

File: components/tree/demo/basic.md [hunk 5]
```diff
@@ -1,10 +1,18 @@
...
+  en-US: basic
 ---
 
+## zh-CN
+
 最简单的用法，展示可勾选，可选中，禁用，默认展开等功能。
 
+## en-US
+
+The most basic usage, tell you how to use checkable, selectable, disabled ,defaultExpandKeys, and etc.
```
Comment (afc163) [inline components/tree/demo/basic.md]: `disabled ,` => `disabled, `

File: components/tree/index.en-US.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+---
+category: Components
+type: Views
+english: Tree
+---
+
+## When to use
+
+Directory, organization , biological classification, country, and etc. Almost things of the world are tree structrue. The `Tree` component is a way of representing the hierachical relationship of these things，and you also can expand, collapse, select the treeNodes of it.
```
Comment (afc163) [inline components/tree/index.en-US.md]: `organization , ` => `organization, `

File: components/tree/index.en-US.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+|multiple | Whether allow to multiple select treeNode | bool | false |
+|checkable | Whether support checkable treeNode | bool   | false    |
+|defaultExpandAll | Whether default to expand all treeNodes | bool | false |
+|defaultExpandedKeys | Specify keys of default expanded treeNodes | String[] | [] |
+|expandedKeys | （controlled）Sepcifies keys of expanded treeNodes | String[] | [] |
+|autoExpandParent | Whether to automatically expand a parent treeNode | bool | true |
+|defaultCheckedKeys | Specifies keys of default checked treeNodes | String[] | [] |
+|checkedKeys | （controlled）Specifies keys of checked treeNodes（PS： When specifies a key of treeNode which is a parent treeNode, all children treeNodes of its will be checked; And vice versa, when specifies a key of treeNode which is a child treeNode, its parent treeNode will also be checked. When `checkable` and `checkStrictly` is true, it'a object has `checked` and `halfChecked` property, and no matter child treeNode or parent treeNode is checked, they won't impact on eachother. | String[]/{checked:Array<String>,halfChecked:Array<String>} | [] |
```
Comment (afc163) [inline components/tree/index.en-US.md]: `（` => `(`

**Comment by benjycui** [comment]
谢谢，或者能再帮忙翻译一些 😅

**Pull request merged**